runtime/parachains/src/inclusion_inherent.rs

```diff
@@ -76,12 +82,12 @@ decl_module! {
 		}
 
 		/// Include backed candidates and bitfields.
-		#[weight = (1_000_000_000, DispatchClass::Mandatory)]
+		#[weight = (INCLUSION_INHERENT_CLAIMED_WEIGHT, DispatchClass::Mandatory)]
```

Comment:
At some point why not just this ? you have access to calls argument in this expression.
```suggestion
		#[weight = (MINIMAL_INCLUSION_INHERENT_WEIGHT + backed_candidates.len() * BACKED_CANDIDATE_WEIGHT, DispatchClass::Mandatory)]
```

Reply:
That doesn't appear to work, unfortunately. It would be a nice solution, but doesn't compile locally.

Reply:
No, I'm wrong: I mis-pasted, which caused the compilation failures. This is a good idea!

Reply:
622f267